cmd/env/command.go

```diff
@@ -36,15 +36,31 @@ command line option                 environment variable
 
 The following configuration options are only available via environment variables:
 %s
+`
+	storageTypeDescription = `The type of backend [%s] used for trace storage.
+Multiple backends can be specified as comma-separated list, e.g. "cassandra,elasticsearch"
+(currently only for writing spans). Note that "kafka" is not a real storage backend, it is
```

Comment:
I'd rephrase this to say "kafka" is distributed queue and not a complete storage solution and is expected to be used in conjunction with ingester.

Reply:
rephrased